test/commands/mode.js

```diff
@@ -20,6 +20,13 @@ describe("Commands", function () {
 			lastCommand: null,
 			nick: "xPaw",
 			irc: {
```

Comment:
Can you add a test for servers without ISUPPORT?

Reply:
Sure thing.